pg_search/src/postgres/build.rs

```diff
@@ -53,6 +53,11 @@ pub extern "C" fn ambuild(
     let index_relation = unsafe { PgRelation::from_pg(indexrel) };
     let index_name = index_relation.name().to_string();
 
+    // Drop the index if it already exists (e.g. if we're rebuilding it with VACUUM FULL)
+    let writer_client = WriterGlobal::client();
+    SearchIndex::drop_index(&writer_client, &index_name)
+        .unwrap_or_else(|err| panic!("error dropping index {index_name}: {err}"));
+
```

Comment:
Doesn't this revert #912 by automatically dropping existing indexes?

Reply:
I don't think so. We have a test for it called `prevent_duplicates` which is passing. This line of code would fire after our checks that prevent a duplicate index from being created.

Reply:
Okay, sounds good!